.cicd/test-package.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+SCRIPT_NAME='test-package.gen.sh'
+
+cat > $SCRIPT_NAME <<EOL
```

Comment:
If we wrap the opening delimiter with single-quotes...
```bash
cat > $SCRIPT_NAME <<'EOL'
```
...then it [won't](https://stackoverflow.com/a/27921346) perform shell interpolation.

While your code is already working and it might suck to change it, I think it will prevent errors for anyone who has to edit it moving forward and forgets an escape sequence. Thoughts?

I, personally, prefer a [recursive solution](https://github.com/EOSIO/eos-vm/blob/615a19214c56473a0508226b5c8cc3aa75b7766f/.cicd/build.sh#L4) because then all the code has syntax highlighting and lives in the same file but I guess I never had time to make the same changes in eos as I did in eos-vm.

Reply:
I'm fine with changing it for readability. I avoid escaping when I can.

Reply:
Hmm, if we do it this way, vim stops highlighting the EOL block though... ;(